session.go

```diff
@@ -120,6 +120,17 @@ func (o *CloseOpts) toMap() map[string]interface{} {
 	return optArgsToMap(o)
 }
 
+// Convenience function that says whether client is still connected
```

Comment:
It would be more conventional to use the following form of comment:
```go
// IsConnected is a function that says whether client is still connected.
```
And I believe names of the `IsSmth` form are against Go's style guides, too.